.readthedocs.yaml

```
@@ -0,0 +1,13 @@
+version: 2
+build:
+  os: ubuntu-lts-latest
+  tools:
+    nodejs: "20"
+  commands:
+    - npm install
+    - npm run build
+    - npm run --workspace packages/mystmd link
+    - myst --version
+    - cd docs && myst build --html
```

```suggestion
    - cd docs && myst build --html --execute
```

Last time it was suggested to use `--execute`, but my understanding is that only affects the running of notebooks. For these issues it should not be relevant right? Is it also used in the mystmd pages?